presto-product-tests/bin/run_on_docker.sh

```diff
@@ -202,7 +221,8 @@ HADOOP_LOGS_PID=$!
 # wait until hadoop processes is started
 # retry check_hadoop
 # stop_unnecessary_hadoop_services
-sleep 300
+sleep 150
```

Comment:
write a `check_mapr` funciton  or make check_hadoop check for mapr srevice if running against a mapr container.